Transfer/Sources/ScheduleTransferSectionData.swift

```diff
@@ -67,14 +67,17 @@ struct ScheduleTransferForeignExchangeData: ScheduleTransferSectionData {
                 let destinationCurrency = foreignExchange.destinationCurrency,
                 let rate = foreignExchange.rate {
                 let sourceAmountFormatted = String(format: "%@ %@",
-                                                   sourceAmount.format(with: sourceCurrency),
+                                                   sourceAmount.formatWithActualDecimal(with: sourceCurrency),
```

Comment:
I'm not sure about the naming of this method `formatWithActualDecimal(with:)`

Reply:
maybe `format(withActualDecimals:)` or `format(similarTo:)`

Reply:
Function name has been changed.